Tutorials/Python_Developer_Tutorials/foundation/RF-Transmit-Receive-Packet/Tutorial_Exp_RF_Packet_TX-User-Input.py

```diff
@@ -1,13 +1,16 @@
+#!/usr/bin/env python
+
 #Imports - General
 
 import os
 import sys
 
-sys.path.append(os.path.join(os.path.dirname(__file__), "../../../../Faraday_Proxy_Tools"))  #Append path to common tutorial FaradayIO module
+# Add Faraday library to the Python path.
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../../..'))
 
-#Imports - Faraday Specific
-from FaradayIO import faradaybasicproxyio
-from FaradayIO import faradaycommands
+# Imports - Faraday Specific
+from faraday.proxyio import faradaybasicproxyio
+from faraday.proxyio import faradaycommands
```

Comment:
Good but I see error unrelated from a prior update that causes a type error: 
![image](https://cloud.githubusercontent.com/assets/339144/25067135/27f7c3aa-21ef-11e7-9669-67090f845822.png)